mongo/integration/mtest/mongotest.go

```diff
@@ -281,8 +282,9 @@ func (t *T) GetStartedEvent() *event.CommandStartedEvent {
 
 // GetSucceededEvent returns the most recent CommandSucceededEvent, or nil if one is not present.
 // This can only be called once per event.
-// TODO(GODRIVER-2075): GetSucceededEvent documents that it returns the most recent event, but
-// actually returns the first event. Update either the documentation or implementation.
+//
+// TODO(GODRIVER-2075): GetSucceededEvent documents that it returns the most recent event, but actually returns the
+// TODO first event. Update either the documentation or implementation.
 func (t *T) GetSucceededEvent() *event.CommandSucceededEvent {
```

Comment:
Same as above, move the TODO comment block to the first line of the `GetSucceededEvent` function.